lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -259,6 +280,11 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
                 raise LisaException(f"file does not exist: {qemu_node_runbook.qcow2}")
 
             vm_disks_dir = os.path.dirname(qemu_node_runbook.qcow2)
+            if self.qemu_platform_runbook.is_host_remote():
+                vm_disks_dir = os.path.join(
+                    "/home", self.qemu_platform_runbook.host.username, vm_name_prefix
```

Comment:
In general it is a bad idea to place VM resources in a user's home directory, as it can cause permissions issues. Perhaps this directory should be a runbook parameter?

Reply:
It's great to make it configurable. In the mean time, it's useful, if there is a default value. So the platform is easier to use.

Reply:
`/srv/` (or `/var/`) might make for a better default.

Reply:
Made it configurable in the runbook and using `/var/tmp` as default.